### PR TITLE
fix:image-responsiveness

### PR DESCRIPTION
**Pull request by anshtripathi01**
**_Please Review_**
fix-image

### CODE REVIEW

File: Components/Images/images.css [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
     font-size: 1rem;
 }
 
 body {
+    text-align: center;
     box-sizing: border-box;
     font-family: 'Roboto', sans-serif;
 }
+.test-class{
+    cursor: pointer !important;
```
Comment (LordUttam) [inline Components/Images/images.css:12]: This style gets overidden by .sidebar-sticky but the cursor there is also a pointer so dont think you really need this marked important. It works the same even without it.

File: Components/Images/images.html [hunk 2]
```diff
@@ -1,23 +1,92 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (LordUttam) [inline Components/Images/images.html:6]: Add author metatag

File: Components/Images/images.css [hunk 3]
```diff
@@ -1,26 +1,32 @@
...
+    margin-left: 5.5rem;
+}
 .responsive-image {
-
-    width: 30rem;
-    height: 20rem;
+    width: 100%;
+    height: auto;
     box-shadow: 0px 0px 5px 2px rgba(0, 0, 0, 0.1);
     border-radius: 0.5rem;
```
Comment (Sharmaryan) [inline Components/Images/images.css:31]: there is no need to write px after 0

**Review by Sharmaryan — APPROVED** [review]

File: Components/Images/images.css (hunk 1 above)
Comment (anshtripathi01) [reply Components/Images/images.css:12]: done

File: Components/Images/images.html (hunk 2 above)
Comment (anshtripathi01) [reply Components/Images/images.html:6]: added author,description meta tag

File: Components/Images/images.css (hunk 3 above)
Comment (anshtripathi01) [reply Components/Images/images.css:31]: done

**Comment by anshtripathi01** [comment]
I am merging this branch

**Pull request merged**